packages/kbn-es-query/src/filters/lib/index.ts

```diff
@@ -93,3 +104,29 @@ export enum FILTERS {
   GEO_BOUNDING_BOX = 'geo_bounding_box',
   GEO_POLYGON = 'geo_polygon',
 }
+
+export const getFilterField = (filter: Filter) => {
```

Comment:
Aside: I was wondering if it would make sense to change this to a `switch` but changed my mind. Having a `switch` would involve more abstraction than what it's worth.